src/components/Profile/Setting.scss

```diff
@@ -0,0 +1,64 @@
+.main {
+  display: 'flex';
+}
+
+.edit-btn {
+  width: 40%;
+  padding: 0.5rem 2rem;
+  margin: 10px auto;
+  border: 1px solid #F34579;
+  background-color: #F34579;
+  border-radius: 25px;
+  display: block;
+  font-size: 1.2rem;
+
+  &:hover {
+    outline: transparent;
+    background-color: #2D6686;
+    color: #FFFFFF;
+  }
+
+  &:focus {
+    outline: transparent;
+  }
+}
+
+textarea {
+  margin: 0;
+  width: 50rem;
+  height: 10rem;
+  -webkit-appearance: none;
+  padding: .78571429rem 1rem;
+  background: #FFF;
+  border: 1px solid rgba(34,36,38,.15);
+  outline: 0;
+  color: rgba(0,0,0,.87);
+  border-radius: .28571429rem;
+  -webkit-box-shadow: 0 0 0 0 transparent inset;
+  box-shadow: 0 0 0 0 transparent inset;
+  -webkit-transition: color .1s ease,border-color .1s ease;
+  transition: color .1s ease,border-color .1s ease;
+  font-size: 1rem;
+  line-height: 1.2857;
+  resize: vertical;
+}
+
+input[type=text] {
```

Comment:
@nadralia, this style needs to be nested in the component class you are trying to style otherwise it will style all input tags with type text. The same applies to the `textarea` tag on line 26